api-reference/task/update-task.mdx

```diff
@@ -0,0 +1,137 @@
+---
+title: "Update Task"
+api: "PATCH https://api.travtus.com/task/"
+description: "This endpoint allows users to update a task created by Adam."
+version: "v2"
+---
+
+This endpoint allows you to update a task with provided details.
+
+You must provide either `identifier` parameter or an external reference via the `external_ref` attribute.
+
+All datetime fields included in the body parameters need to be specified as [ISO8601-formatted date strings](https://en.wikipedia.org/wiki/ISO_8601).
+
+Datetime fields should follow a "YYYY-MM-DDThh:mm:ss" format.
+
+### Header
+
+<ParamField header="Authorization" type="string" default="none" required>
+    The authentication token for your request
+</ParamField>
+
+### Body
+
+<ParamField body="identifier" type="string" default="none">
+    The identifier of a task to use for updating a task.
+
+    Please note you can either use the `identifier` or `external_ref` to update a task.
+</ParamField>
+
+<ParamField body="external_ref" type="string" default="none">
+    The external reference of a task to use for updating a task.
+
+    Please note you can either use the `identifier` or `external_ref` to update a task.
+</ParamField>
+
+<ParamField body="status" type="enum" default="none">
+    The status of a task.
+
+    Possible value: `created`, `in_progress`, `completed`, `deleted`
+</ParamField>
+
+<ParamField body="due_datetime" type="datetime" default="none">
+    The due date of a task. Example: 2023-01-01T00:00:00
+</ParamField>
+
+<ParamField body="reminder_datetime" type="string" default="none">
+    The reminder date of a task. Example: 2023-01-01T00:00:00
```

Comment:
```suggestion
    The reminder date of the task. Example: 2023-01-01T00:00:00
```
Also for all of the following ones.

Alternatively, you can simply say `The task reminder date` - no need for an a/the article.